linode/placementgroup/framework_datasource_schema.go

```diff
@@ -47,5 +55,22 @@ var DataSourceSchema = schema.Schema{
 				},
 			},
 		},
+		"migrations": schema.ListNestedBlock{
+			Description: "A list of migration objects associated with the placement group. Empty if no migrations exist.",
+			NestedObject: schema.NestedBlockObject{
+				Attributes: map[string]schema.Attribute{
+					"inbound": schema.ListAttribute{
+						Description: "The compute instances being migrated into the placement group.",
+						Computed:    true,
+						ElementType: pgMigrationInstanceObjectType,
+					},
+					"outbound": schema.ListAttribute{
+						Description: "The compute instances being migrated out of the placement group.",
+						Computed:    true,
+						ElementType: pgMigrationInstanceObjectType,
+					},
+				},
+			},
+		},
```

Comment:
Since it not actually a list, can we use `SingleNestedBlock` rather than `ListNestedBlock` for it?

Reply:
Good point. Updated to use `SingleNestedBlock`. Thank you!